lib/const.rb

```diff
@@ -1,5 +1,5 @@
 # Defines common constants used in different parts of crew
-CREW_VERSION = '1.35.3'
+CREW_VERSION = '1.35.4'
```

Comment:
This needs to be synced with master because we're currently on `1.35.4` now.

Reply:
Fixed.